FEATURES.md

```diff
@@ -0,0 +1,87 @@
+
+From this point, the maintainers will engage you on the feature proposal in the issue comments. The goal of this conversation is to clarify if the feature proposal meets the criteria of a new feature in the project. An additional goal is to determine if this is something that you, as the submitter, can contribute yourself or if you’ll need assistance from additional contributors.
+
+The criteria to continue with a feature proposal are as follows:
+
+- __Is the proposal describing a feature?__
+ The feature proposal isn’t meant for bug fixes, refactoring, or refinement to existing features. Sometimes this line can be fuzzy so the maintainers will help clarify.
+- __Does this feature fit in the scope of the repo and project?__
+Some things might be good ideas but they might not fit either with the project as a whole in the specific repo in which the issue was posted. In the latter case, the maintainers will help you determine the right home and transfer the issue.
+- __Is the proposed feature sufficiently distinct from existing features?__
+With a project like OpenSearch it’s totally normal to not know every capability, so this criteria is designed to suss out if the feature provides unique capabilities when compared to existing functionality.
+- __Does the proposed feature overlap or conflict with ongoing work?__
+The best use of everyone’s time is to ensure that there are no competing implementations of features, so this criteria establishes that there is not more than one effort going on for any given area.
+- __Is the feature proposal sufficiently atomic as a feature?__
+Sometimes big ideas should be broken up into smaller ideas. Complete atomicity is probably unobtainable given interdependencies between different facets of the product however the goal here is to make sure that the proposed features are accomplishable in a reasonable amount of time or if it would make more sense to break up the feature into smaller chunks.
+
+The maintainers of the repo decide when and if the proposal meets the criteria. If the feature proposal doesn’t meet the criteria, then the issue will be closed with an explanation and tagged clearly. If the feature does meet the criteria and someone is volunteering to work on the feature, it moves to the next stage. If the feature still needs someone to write the feature, it is tagged as “help wanted” until someone volunteers.
+
+In this process your issue might be closed for a variety of reasons and it doesn't necessarily mean that it’s a bad idea or that the feature isn’t wanted. It might mean that the way it is presented, supported, or the context of the other development in the project doesn’t align with the feature proposal process. If this is the case, the maintainer can help with next steps to move forward.
+
+__Creating META issue(s):__
```

Comment:
Make this a `######...` section? It should be meta-issue, META is a company ticker.